requirements.txt

```diff
@@ -17,7 +17,4 @@ arch==4.19
 requests==2.25.1
 statsmodels==0.12.2
 xarray==0.18.0
-quandl>=3.6.1,<=3.7.0
-
---extra-index-url https://bcms.bloomberg.com/pip/simple/
-blpapi>=3.16.2,<3.18.0
```

Comment:
I think we need blpapi, because the pipeline fails with the errors 

`ModuleNotFoundError: No module named 'blpapi'`